### PR TITLE
Revert "[INTERNAL] Move Transfer Listener Logic"

### PR DESCRIPTION
**Pull request by m273d15**
Reverts saros-project/saros#127

@srossbach still required changes on this PR. Therefore we have to revert this PR

### CODE REVIEW

**Comment by m273d15** [comment]
@stefaus  Are there PRs (which are already merged) that base on this PR?

**Comment by stefaus** [comment]
yes, merged #128 and open as follow-up #154, #155, #156, #157, #159

File: de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/ArchiveIncomingProjectNegotiation.java [hunk 1]
```diff
@@ -101,16 +117,34 @@ protected void transfer(IProgressMonitor monitor,
...
+            receiveAndUnpackArchive(projectMapping, archiveTransferListener,
+                monitor);
         }
     }
 
+    @Override
+    protected void cleanup(IProgressMonitor monitor,
+        Map<String, IProject> projectMapping) {
+        /*
+         * TODO Move disable queuing responsibility to SarosSession (see todo
```
Comment (saros-infrastructure) [inline de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/ArchiveIncomingProjectNegotiation.java]: Code Smell Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=160&issues=AWTLb7qM6dA5b-W27MG0&open=AWTLb7qM6dA5b-W27MG0)

[ISSUE_KEY]: AWTLb7qM6dA5b-W27MG0

File: de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/ArchiveIncomingProjectNegotiation.java [hunk 2]
```diff
@@ -61,6 +64,19 @@ public ArchiveIncomingProjectNegotiation(
...
     }
 
+    @Override
+    protected void setup(IProgressMonitor monitor)
+        throws IOException {
+        archiveTransferListener = new ArchiveTransferListener(
+            ARCHIVE_TRANSFER_ID + getID());
+
+        if (fileTransferManager == null)
+            // FIXME: the logic will try to send this to the remote contact
```
Comment (saros-infrastructure) [inline de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/ArchiveIncomingProjectNegotiation.java]: Code Smell Code Smell: Take the required action to fix the issue indicated by this comment. ([squid:S1134](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS1134))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=160&issues=AWTLb7qM6dA5b-W27MG1&open=AWTLb7qM6dA5b-W27MG1)

[ISSUE_KEY]: AWTLb7qM6dA5b-W27MG1

**Pull request merged**